mcsema/BC/Function.cpp

```diff
@@ -155,6 +158,17 @@ static llvm::Function *GetLiftedFunction(const NativeModule *cfg_module,
   return nullptr;
 }
 
+static llvm::Function *GetExceptionHandlerPrologue(void) {
```

Comment:
Brief comment before function describing what the exception handler prologue function does, and that it can be found in the ABI-specific printed runtime file.